trace-dispatcher/src/Cardano/Logging/Tracer/Standard.hs

```diff
@@ -71,14 +71,16 @@ standardTracer = do
 startStdoutThread :: IORef StandardTracerState -> IO ()
 startStdoutThread stateRef = do
     (inChan, outChan) <- newChan 2048
-    as <- async (stdoutThread outChan)
+    as <- async (catch
+      (stdoutThread outChan)
+      (\(_ :: BlockedIndefinitelyOnMVar) -> pure ()))
     link as
     modifyIORef stateRef (\ st ->
```

Comment:
```suggestion
    modifyIORef' stateRef (\ st ->
```

Reply:
Or probably `atomicModifyIORef'` would be even better.

Reply:
Done